contentful/apps
# feat: add notification for user when copying to clipboard

## Purpose

The purpose of this PR is to add a small notification for the user when they copy something to the clipboard. 

## Approach
I created a hook, `useNotifier`, to encapsulated this copy notification logic, as we can use the same hook for other notification as we see fit (and can expand the return value of the hook). With introducing this hook, it utilizes the App `SDK`, so I had to subsequently mock the `SDK` in tests and add the `SDKProvider` to the modal itself. 

![Screenshot 2024-04-10 at 8 06 26 AM](https://github.com/contentful/apps/assets/58186851/a2be264d-8829-4420-859b-2c77e7b6d833)

## Testing steps

Copy one of the "copy-able" parts of the config page and notice the notification appear!
